demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -139,6 +139,142 @@ class DemoTileOrganizer {
   }
 }
 
+class DemoVideoTile extends HTMLElement {
```

Comment:
[I don't know why we keep everything in a 3.5k line file and at this point I'm too afraid to ask](https://i.kym-cdn.com/entries/icons/original/000/016/724/prattypratprat.PNG)

Reply:
We can move it out though right similar to video filter processor

Reply:
We could move it to a separate one like video filter right?

Reply:
I'll save it for another PR.  It feels a bit strange to just have this in its own file given all the other tile logic is either in this file or the CSS :P.